src/entity/user/UserComment.ts

```diff
@@ -10,15 +10,20 @@ import {
 import type { User } from './User';
 import type { Comment } from '../Comment';
 import { UserVote } from '../../types';
+import type { UserTransaction } from './UserTransaction';
 
 export type UserCommentFlags = Partial<{
-  // flags
+  awardId: string;
 }>;
 
 @Entity()
 @Index(['commentId', 'userId'], { unique: true })
 @Index(['userId', 'vote', 'votedAt'])
 export class UserComment {
+  get awarded(): boolean {
+    return !!this.awardTransactionId;
+  }
```

Comment:
Just check that this works, had some issues in the past with it due to typeorm layer on top.

Reply:
yep, it is used in graphorm, it could be this or in graphorm do `{ ...entity, awarded: !!entity.awardTransactionId }`